jupyter-notebooks/notebooks/Example Platform-library PySpark Notebook.ipynb

```diff
@@ -506,7 +512,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython2",
-   "version": "2.7.6"
+   "version": "2.7.14"
```

Comment:
What's this determined by? Candidate for Pillar.

Reply:
This is an auto-generated field by jupyter runtime environment (i.e. pyspark kernel). In this particular case, pyspark's venv is set to /opt/pnda/anaconda/bin/python, which is python 2.7.14. (see screenshot). 
<img width="705" alt="screen shot 2018-05-21 at 12 04 54" src="https://user-images.githubusercontent.com/7880763/40304591-40742d92-5cef-11e8-9edd-99b77fb3c684.png">